### PR TITLE
Feature/remove hero bound abilities

### PR DESCRIPTION
**Pull request by Surxe**
Removes duplicate data from BoundAbilities in hero data that is sufficiently in ability data instead. The only remaining keys are Key and Name(english).

_Parsed data in [deadlock-data PR](https://github.com/deadlock-wiki/deadlock-data/pull/15)_

### CODE REVIEW

File: src/parser/parser.py [hunk 1]
```diff
@@ -141,8 +142,20 @@ def _parse_heroes(self, parsed_abilities):
             json_utils.sort_dict(parsed_meaningful_stats),
         )
 
+        stripped_heroes = dict()
+        # Remove irrelevant data from BoundAbilities in HeroData
+        for hero_key, hero_value in copy.deepcopy(parsed_heroes).items():
```
Comment (hassayag) [inline src/parser/parser.py:147]: looks like hero_value is not being reassigned anywhere, so is a copy strictly necessary?

Comment (Surxe) [reply src/parser/parser.py:147]: I tried without first; if you set a dict value within stripped_heroes to be the dict value from parsed_heroes, they point to the same address, causing the nested values within BoundAbilities to be shared. Couldn't get it to work without the deepcopy. Lmk if you can think of a cleaner solution though

Comment (hassayag) [reply src/parser/parser.py:147]: Oh yeah it assigns it to stripped heroes, which is then manipulated. Thats fine then

**Review by hassayag — APPROVED** [review]
Looks good, just make sure the wiki doesn't rely on this data

**Pull request merged**